gluten-ut/spark35/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/GlutenParquetFieldIdIOSuite.scala

```diff
@@ -16,6 +16,27 @@
  */
 package org.apache.spark.sql.execution.datasources.parquet
 
-import org.apache.spark.sql.GlutenSQLTestsBaseTrait
+import org.apache.spark.sql.{GlutenSQLTestsBaseTrait, Row}
 
-class GlutenParquetFieldIdIOSuite extends ParquetFieldIdIOSuite with GlutenSQLTestsBaseTrait {}
+class GlutenParquetFieldIdIOSuite extends ParquetFieldIdIOSuite with GlutenSQLTestsBaseTrait {
```

Comment:
@weixiuli Do we need to add this test for spark 32 33 and 34?

Reply:
> @weixiuli Do we need to add this test for spark 32 33 and 34?

 Thanks, done. Add UT for 33 and 34, 32 doesn't have WriteFilesExec, so it doesn't need to add. BTW, the existing unit tests in spark 32, 33, 34 ,35 also can cover this change.